schema_based_on_diagram.sql

```diff
@@ -0,0 +1,52 @@
+CREATE TABLE patients{
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(50),
+    date_of_birth DATE NOT NULL
+}
```

Comment:
- [ ] I could not create any of your tables because I got a syntax error. it happened that you use `{ }` instead of `( )` for creating the table. Let's fix this error by replacing `{ }` with `( )`. And please do not forget to add `;` at the end so that we can successfully create all your tables. *Kindly fix this error in all your tables*

```suggestion
CREATE TABLE patients (
    id SERIAL PRIMARY KEY,
    name VARCHAR(50),
    date_of_birth DATE NOT NULL
);
```

See the error message below;
<img width="449" alt="Code_mAXhHEerkX" src="https://user-images.githubusercontent.com/81468675/166716247-97a94d34-f7ba-4e52-a5d9-89dbcab26fae.png">


See when I fix the error by replacing `{ }` with `( )`;
<img width="348" alt="Code_5pf2ZXNEcT" src="https://user-images.githubusercontent.com/81468675/166716548-4334d0d0-5b21-4c6c-85e0-0647d03195de.png">